[StyleCop] Fix all warnings in Japanese/MergedDateTimeExtractor file

ceciliaavila opened the pull request:
- Renamed file to 'JapaneseMergedExtractor' to match the class name.

JuanAr reviewed (changes requested):
If you add the `&w=1` (Or using the PR diff settings drop down) query string to this PR you'll see only a few lines are modified. However, the entire file seems modified in this PR. This is due to the VS warning regarding mixed new lines.
Keep only the minimum changes needed to fix the SA warnings.

Bill7zz committed:
Fix Stylecop warning rename file to JapaneseMergedExtractor

Bill7zz committed:
Fix StyleCop warnings in JapaneseMergedExtractor

-Reorder using statement
-Remove extra blank line
-Fix variable name with Hungarian notation

Bill7zz reviewed (approved):
LGTM
